mk/docker.mk

```diff
@@ -89,7 +96,7 @@ images: images/release images/test
 images/release: image/kuma-cp image/kuma-dp image/kumactl image/kuma-init image/kuma-prometheus-sd ## Dev: Rebuild all Docker images
 
 .PHONY: images/test
-images/test: image/kuma-universal
+images/test: image/kuma-universal docker/build/postgres
```

Comment:
Nit: might be preferable to add an empty `image/postgres` target just so this is consistent with the pattern of targets.